src/test/resources/config/config.yaml

```diff
@@ -0,0 +1,23 @@
+server:
```

Comment:
why do we need this file all of a sudden?

Reply:
Running the integration tests under the failsafe plugin leads to an exception being thrown by the dropwizard resourcehelper:

`java.lang.RuntimeException: java.lang.IllegalArgumentException: URI is not hierarchical`

I don't really know why - presumably it's because it sets up a strange classpath?